kitchen/modules/execution-schedule/README.md

```diff
@@ -0,0 +1,51 @@
+# Execution Schedule
+
+This module abstracts away the scheduling of task execution and governs the priority of tasks using a council of `AccountId`s stored in runtime storage. 
+
+Although we could add more nuanced governance to the `Council`, the purpose of this example is to demonstrate how and when to use `on_initialize` and `on_finalize`.
+
+This example also demonstrates
+* generous usage of type aliases
+* sorting tasks by priority in `on_finalize`
+* signalling priority by council in the runtime (`signal_priority`)
+* preparing on-chain state in `on_initialize`
+* using a `double_map`
+* unit testing with a mock runtime
+
+The context of the example is scheduling the execution of tasks in a queue stored in the runtime. Tasks are structs declared like
+
+```rust
+pub struct Task<BlockNumber> {
+    id: TaskId,
+    priority_score: PriorityScore,
+    proposed_at: Timestamp,
+    expected_execution: BlockNumber,
+}
+```
+
+# stuck
+
+Right now, it isn't compiling -- I'm getting this error and I believe it's hiding many more errors. I've been refactoring code around it but I need to compile at some point. the path forward might be rewriting it from scratch and incrementally compiling because I tested the syntax used in `balances` for the `VestingSchedule` in `struct-storage` and it worked perfectly.
```

Comment:
basically, doesn't work with `add_extra_genesis` but the most recent commit gets compilation and test passage because it doesn't rely on genesis build instructions from `decl_storage`.

It would be nice if I could add more logic to express what I want runtime storage to be initialized to at the start, it is not *necessary*. The alternative that I've been considering is just adding a `impl<T: Trait> Module<T> {}` with a few methods to change mock runtime storage for the mock runtime unit tests.

Reply:
I implemented additional methods to configure storage in the test submodule as per the above commend. The tests are satisfactory.